core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -75,6 +77,34 @@ public static int getProcessId() {
     return PosixLoader.POSIX.getpid();
   }
 
+  /**
+   * Returns {@code true} if JNR {@link Platform} class is loaded, and {@code false} otherwise.
+   *
+   * @return {@code true} if JNR {@link Platform} class is loaded.
+   */
+  public static boolean isPlatformAvailable() {
+    try {
+      return PlatformLoader.PLATFORM != null;
+    } catch (NoClassDefFoundError e) {
+      return false;
+    }
+  }
+
+  /**
+   * Returns the current processor architecture the JVM is running on, as reported by {@link
+   * Platform#getCPU()}.
+   *
+   * @return the current processor architecture.
+   * @throws UnsupportedOperationException if JNR Platform library is not loaded.
+   */
+  public static String getCPU() {
+    if (!isPlatformAvailable())
+      throw new IllegalStateException(
+          "JNR Platform class not loaded. "
+              + "Check isGetProcessIdAvailable() before calling this method.");
```

Comment:
Should be: "Check isPlatformAvailable()".

Reply:
done, sorry for that copy/pasting